README.md

```diff
@@ -61,6 +61,10 @@ You can [sign up for a free account](https://app.posthog.com/signup) on our host
 
 [See docs for developing locally](https://posthog.com/docs/developing-locally)
 
+Don't want to install or run anything locally? Develop remotely on Porter.
```

Comment:
I think this would be better placed in the linked guide. You can create a PR for that [here](https://github.com/PostHog/posthog.com/blob/master/contents/docs/developing-locally.md)

Reply:
We removed the instructions in the main README.md for this PR and opened a PR on the linked guide instead - this PR should be ready to go!